src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -3769,9 +3769,11 @@ def spawn(self, launcher, cu):
         with open(launch_script_name, "w") as launch_script:
             launch_script.write('#!/bin/bash -l\n\n')
 
-            launch_script.write("echo script start_script `%s` >> %s/PROF\n" % (cu['gtod'], cu_tmpdir))
+            if 'RADICAL_PILOT_PROFILE' in os.environ:
```

Comment:
Good question.  No, we don't have a flag.  We may want to consider adding the value to the agent config, which is the canonical place for these things anyway.

Reach:  it seemed to  work on BW, avoiding the `PROF` parsing errors when profiling was disabled.  I am not perfectly sure if env is forwarded on all agent launch methods.